src/main/java/tohru/command/Parser.java

```diff
@@ -0,0 +1,72 @@
+package tohru.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import tohru.exception.TohruException;
+
+/**
+ * Parser represents the parsing of input prompt and determine which command to execute
+ */
+public class Parser {
+
+    /**
+     * Parse the giving input and determine the corresponding commands
+     *
+     * @param prompt The input from the user
+     * @return A command object representing the operation requested by the user
+     * @throws TohruException When an invalid command is passed in
+     */
+    public static Command parse(String prompt) throws TohruException {
+
+        String[] dissectedPrompt = prompt.trim().split(" ", 2);
+        String command = dissectedPrompt[0];
+        String arguments = null;
+        if (dissectedPrompt.length == 2) {
+            arguments = dissectedPrompt[1].trim();
+        }
+
+        switch (command) {
```

Comment:
Nice Egyptian Brace and Good Indentation for Switch Case. Keep up the good work.